test-harness/src/test/java/com/netflix/conductor/tests/utils/MockExternalPayloadStorage.java

```diff
@@ -102,6 +102,7 @@ public InputStream download(String path) {
                     InputStream opStream = MockExternalPayloadStorage.class.getResourceAsStream("/output.json");
                     return objectMapper.readValue(opStream, Map.class);
                 case INPUT_PAYLOAD_PATH:
+                case "workflow/output":
```

Comment:
Nit: Use a static variable as in L#104 to keep it consistent.

Reply:
done